dom.bs

```diff
@@ -6929,7 +6939,8 @@ are:
 <div algorithm>
 <p>To <dfn id=concept-attach-a-shadow-root>attach a shadow root</dfn>, given an
 <a for=/>element</a> <var>element</var>, a string <var>mode</var>, a boolean <var>clonable</var>,
-a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
+a boolean <var>serializable</var>, a boolean <var>delegatesFocus</var>, and a boolean
+<var>slotAssignment</var>:
```

Comment:
_slotAssignment_ is a string. Rebasing will likely correct this.

Reply:
Done, and it did.